docs/portal/portal-overview.md

```diff
@@ -37,7 +37,11 @@ There are two main ways to access the pages, as shown in the following image:
 
 ![UID2 Portal Structure](images/portal-uid2-overview.png)
 
-Your participant name will be displayed at the top of the left sidebar. If your account has access to multiple participants, as shown in the example above, a dropdown menu will appear. You can use this menu to switch between participants in the portal. 
+The name of the current sharing participant is displayed at the top of the left sidebar. If your account has access to multiple participants, you'll see a drop-down list of the participants that you have access to. You can use this to switch between participants.
```

Comment:
I don't think 'sharing participant' is the right term here, just 'participant' or 'selected partcipant'

Reply:
Removed that, thx.